client/my-sites/sidebar/static-data/domain-only-fallback-menu.ts

```diff
@@ -0,0 +1,36 @@
+import { translate } from 'i18n-calypso';
+
+const domainOnlyFallbackMenu = ( { siteDomain }: { siteDomain: string } ) => {
+	return [
+		{
+			icon: 'dashicons-admin-settings',
+			slug: 'manage-domain',
+			title: translate( 'Manage domain' ),
+			type: 'menu-item',
+			url: `/domains/manage/${ siteDomain }/edit/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-admin-settings',
+			slug: 'manage-email',
+			title: translate( 'Manage email' ),
+			type: 'menu-item',
+			url: `/email/${ siteDomain }/manage/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-cart',
+			slug: 'manage-purchases',
+			title: translate( 'Manage purchases' ),
+			type: 'menu-item',
+			url: `/purchases/subscriptions/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-email',
+			slug: 'inbox',
+			title: translate( 'Inbox' ),
+			type: 'menu-item',
+			url: `/inbox/${ siteDomain }`,
+		},
```

Comment:
For domain-only sites without email, I'm wondering if we should hide this..? 

I know that we don't do this for regular sites, but in this case, users will basically get identical views if they click "Manage email" or if they click "Inbox".

Reply:
We can hide the "Manage email" menu item instead of "Inbox", since "Manage email" seems to suggest that the user has an email subscription that needs to be managed. And we leave "Inbox" in place as an upsell.

Reply:
The "Manage email" item was hidden, based on feedback from pbLl1t-1f6-p2#comment-1265